tests/test_offchain_types.py

```diff
@@ -326,29 +328,14 @@ def test_invalid_object():
     assert e.value.code == "invalid_object"
 
 
-def test_field_value_because_of_invalid_object_type_for_union_type():
+def test_field_value_because_of_invalid_object_type_for_command():
     request_json = """{
   "cid": "3185027f-0574-6f55-2668-3a38fdb5de98",
   "command_type": "PaymentCommand",
   "command": "invalid",
   "_ObjectType": "CommandRequestObject"
 }"""
-    with pytest.raises(offchain.FieldError, match="expect json object, but got str: ") as e:
-        offchain.from_json(request_json)
-
-    assert e.value.code == "invalid_field_value"
-
-
-def test_invalid_object_type():
```

Comment:
removed this test because json deserialization does not validate command_type is matching with command._ObjectType anymore, the validation is moved to offchain.client, or user's code, as we don't deserializing the command object when deserializing the request object.